release.sh

```diff
@@ -1,16 +1,95 @@
 #!/bin/bash
 
+# Builds and uploads a release, optionally creating a tag for that release.
```

Comment:
Perhaps we can say what we're releaseing explicitly.  I think it's the SDK, but we have services in the repo.